src/uwtools/cli.py

```diff
@@ -238,11 +234,7 @@ def _dispatch_config_validate(args: Args) -> bool:
 
     :param args: Parsed command-line args.
     """
-    return uwtools.api.config.validate(
-        input_file=args[STR.infile],
-        input_format=args[STR.infmt],
-        schema_file=args[STR.schemafile],
-    )
+    return uwtools.api.config.validate(schema_file=args[STR.schemafile], config=args[STR.config])
```

Comment:
Any reason for the order of args here?

Reply:
This order matches the signature of `uwtools.api.config.validate()`, which matches the signature of `uwtools.config.validator.validate_yaml()`, where the order matters because `schema_file` is required and `config` is optional. (I'd accidentally lost the `= None` default on `config` but just added it back -- so I'm glad you asked!)